homeassistant/components/nuki/__init__.py

```diff
@@ -188,7 +188,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             entry.data[CONF_HOST],
             entry.data[CONF_TOKEN],
             entry.data[CONF_PORT],
-            True,
+            entry.data[CONF_ENCRYPT_TOKEN],
```

Comment:
As you don't create any migration, this will raise an `KeyError` for existing config entries.
I suggest:
```suggestion
            entry.data.get(CONF_ENCRYPT_TOKEN, True),
```

Reply:
Oh, yeah. good point!

Reply:
Thanks, implemented this, now the tests are also green again, as there is no migration required :)